dragn/dice/__init__.py

```diff
@@ -1,6 +1,22 @@
+import re
+
 from dragn.dice.die import DieBuilder
 
 
+# Module level __getattr__ to automatically create dice when
+# one a die of non-standard sizes.
+def __getattr__(name: str):
+    m = re.match("D(\d+)", name)
+    # Ovoiding te Walrus operator here for backwards compatibility :(
+    if name is None:
```

Comment:
If we place this guard earlier so we don't need to execute the regex match if `name is None`.

Reply:
This was supposed to be a check that the match is not None. I shall make appropriate adjustments.